discovery/ecs/ecs_i_test.go

```diff
@@ -0,0 +1,118 @@
+// +build integration
```

Comment:
It's because these tests are integration tests (not AWS integration, that requieres AWS accounts, that's why so many mocks) but prometheus SD integration test as a whole, not small unit tests like the other ones